pkg/sink/sink.go

```diff
@@ -235,10 +234,12 @@ func (r Sink) HandleEvent(response http.ResponseWriter, request *http.Request) {
 
 	} else {
 		responseEvent := cloudevents.NewEvent()
-		responseEvent.SetID(uuid.New().String())
-		responseEvent.SetSource(r.EventListenerName)
+		responseEvent.SetID(elUID)
+		responseEvent.SetSubject(r.EventListenerNamespace + "." + r.EventListenerName + " processing " + elUID)
+		responseEvent.SetSource(r.EventListenerName + "." + r.EventListenerNamespace)
```

Comment:
This looks all right. Alternatively it could be `elUID`.

Reply:
Actually the source is little bit more complicated:
See: https://github.com/tektoncd/triggers/pull/1600#issuecomment-1569007791